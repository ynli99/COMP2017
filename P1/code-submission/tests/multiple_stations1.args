Sydney Melbourne "19:40:59"
