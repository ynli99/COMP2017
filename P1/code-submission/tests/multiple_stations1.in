Sydney::Melbourne::08:05:00
Sydney::Melbourne::09:32:45
Sydney::Melbourne::11:45:32
Sydney::Melbourne::13:12:19
Sydney::Melbourne::19:59:15
Sydney::Melbourne::22:32:21
Melbourne::Sydney::14:13:12
Melbourne::Sydney::16:55:12
Melbourne::Sydney::18:33:12
Melbourne::Sydney::20:25:12
Melbourne::Sydney::22:22:12
