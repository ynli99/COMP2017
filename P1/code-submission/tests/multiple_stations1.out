The next train to Melbourne from Sydney departs at 19:59:15
